tutorials/variational-inference/01_variational-inference.jmd

```diff
@@ -190,124 +116,66 @@ And one significant advantage of using `vi` is that we can sample from the resul
 q isa MultivariateDistribution
 ```
 
-
-
-
-    true
-
-
-
 This means that we can call `rand` to sample from the variational posterior `q`
 
 
 ```julia
 rand(q)
 ```
 
-
-
-
-    2-element Array{Float64,1}:
-      1.0134702063474585
-     -0.07429020521027016
-
-
-
 and `logpdf` to compute the log-probability
 
 
 ```julia
 logpdf(q, rand(q))
 ```
 
-
-
-
-    4.277478745320889
-
-
-
 Let's check the first and second moments of the data to see how our approximation compares to the point-estimates form the data:
 
 
 ```julia
 var(x), mean(x)
 ```
 
-
-
-
-    (1.021109459575047, -0.028838703049547422)
-
-
-
-
 ```julia
 (mean(rand(q, 1000); dims = 2)..., )
 ```
 
-
-
-
-    (1.02716749432684, -0.02510701319723139)
-
-
-
 That's pretty close! But we're Bayesian so we're not interested in *just* matching the mean.
 Let's instead look the actual density `q`. 
 
 For that we need samples:
 
 
-```julia
+```julia; results = "hidden"
 samples = rand(q, 10000);
 ```
 
-
 ```julia
 # setup for plotting
 using Plots, LaTeXStrings, StatsPlots
 pyplot()
 ```
 
-    ┌ Info: Precompiling PyPlot [d330b81b-6aea-500a-939a-2ce795aea3ee]
-    └ @ Base loading.jl:1260
-
-
-
-
-
-    Plots.PyPlotBackend()
-
-
-
-
 ```julia
 p1 = histogram(samples[1, :], bins=100, normed=true, alpha=0.2, color = :blue, label = "")
 density!(samples[1, :], label = "s (ADVI)", color = :blue, linewidth = 2)
-density!(collect(skipmissing(samples_nuts[:s].value)), label = "s (NUTS)", color = :green, linewidth = 2)
+density!(collect(skipmissing(samples_nuts[:s].data)), label = "s (NUTS)", color = :green, linewidth = 2)
```

Comment:
Is this change needed? IMO probably we should use something different here since we have all the array-conversion methods.

Reply:
Why can't we just state the parameter BTW?